rotkehlchen/db/optimismtx.py

```diff
@@ -70,6 +107,6 @@ def _build_evm_transaction(self, result: tuple[Any, ...]) -> OptimismTransaction
             gas_used=int(result[9]),
             input_data=result[10],
             nonce=result[11],
-            l1_fee=0 if result[12] is None else int(result[12]),
+            l1_fee=0 if result[12] is None else int(result[12]),  # this check is only needed when _build_evm_transaction is called from the get_evm_transactions method.  # noqa: E501
```

Comment:
Since this may change in the future have a more specific comment

```suggestion
            l1_fee=0 if result[12] is None else int(result[12]),  # this check is only needed when _build_evm_transaction is called from a code path that does not call assert_tx_data_is_pulled().  # noqa: E501
```